app/views/static_pages/team.html.erb

```diff
@@ -51,6 +51,9 @@
                     <%= link_to image_tag("team/patrizia.jpeg", class: "circle"), "https://www.linkedin.com/in/patrizia-trammell/", target: "_blank" %>
                     <div class="name pt-2">Patrizia Trammell</div>
                     <div class="member-title">Tacoma Chapter Lead</div>
+                    <div class="tooltip-description">
+                            Patrizia has experience in Social Media, Customer Service and 15+ years Management and now transitioned into a Full-stack Web and Mobile Developer. 
+                        </div>
```

Comment:
@PatriziaT you will need to wrap this into the tooltip to make sure this text appears in the tooltip on hover event.

Reply:
```
<div class="the-tooltip">
    <div class="name pt-2">Patrizia Trammell</div>
    <div class="member-title">Tacoma Chapter Lead</div>
    <div class="tooltip-description">
        Patrizia has experience in Social Media, Customer Service and 15+ years Management and now transitioned into a Full-stack Web and Mobile Developer. 
    </div>
</div>
```

Reply:
Make sure it renders ok when you run it locally! Thanks :)